src/node.cc

```diff
@@ -4188,6 +4188,11 @@ void Init(int* argc,
   V8::SetFlagsFromString(NODE_V8_OPTIONS, sizeof(NODE_V8_OPTIONS) - 1);
 #endif
 
+  // Allow for environment set preserving symlinks.
+  if (auto preserve_symlinks = secure_getenv("NODE_PRESERVE_SYMLINKS")) {
+     config_preserve_symlinks = (*preserve_symlinks != '\0');
```

Comment:
Indentation seems to be off here and I'm having second thoughts about interpreting `NODE_PRESERVE_SYMLINKS=0` as "yes please, enable this feature."  Perhaps a more robust atoi or strtol-based parser is in order.